R/quarto.R

```diff
@@ -19,9 +19,9 @@ quarto_path <- function() {
 find_quarto <- function() {
   path <- quarto_path()
   if (is.null(path)) {
-    stop("Unable to find quarto command line tools.")
+    stop("Quarto command-line tools path not found! Please make sure you have installed and added Quarto to your PATH.")
```

Comment:
Maybe we can add mention of `QUARTO_PATH` too . This env var is used in `quarto_path()` and can be used for specifying path to quarto without touching the PATH 
https://github.com/quarto-dev/quarto-r/blob/36075964f6bc11694b5fe16d83c6e7085c6d26a1/R/quarto.R#L10